ci/jenkins/utils.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function check_and_upgrade_golang() {
+    export GOROOT=/usr/local/go
+    export PATH=${GOROOT}/bin:$PATH
+    echo "====== Checking installed Golang version ======"
+    golang_minor_version=$(go mod edit -json | jq -r .Go)
+    antrea_golang_version="go$(go mod edit -json | jq -r .Go)"
+    installed_version=$(/usr/local/go-${golang_minor_version}/bin/go version | grep -oP "go\d+\.\d+(\.\d+)?") || true
+    # Since the antrea go.mod file does not contain the golang patch number, we will query go.dev
+    # and use the latest patch number.
+    sudo apt update && sudo apt install -y curl
```

Comment:
can we avoid doing this here? Can we just assume that curl is always installed on the testbeds, and do it manually if it's not the case?

Reply:
Yes we can. The `curl` is installed on the testbed by default, removed this line.